locale/es.json

```
@@ -343,52 +343,52 @@
 	"pages.administration.ldapEdit.roles.placeholder.user": "Valor del atributo ignorar usuari@",
 	"pages.administration.ldapEdit.validation.path": "Por favor, utilice un formato de ruta LDAP",
 	"pages.administration.ldapEdit.validation.url": "Por favor, utilice un formato de URL válido",
-	"pages.administration.migration.back": "Zurück",
-	"pages.administration.migration.cannotStart": "Migration kann nicht beginnen. Die Schule befindet sich nicht im Migrationsmodus oder in der Tranfer-Phase.",
+	"pages.administration.migration.back": "Paso anterior",
+	"pages.administration.migration.cannotStart": "La migración no puede comenzar. La escuela no está en modo de migración ni en fase de traslado.",
```

Maybe we should use the same wording for transfer phase like in pages.administration.migration.finishTransferPhase(line 353) and pages.administration.migration.step4 (line 360)?

```suggestion
	"pages.administration.migration.cannotStart": "La migración no puede comenzar. La escuela no está en modo de migración ni en fase de transferencia.",
```